components/resources/ResourceBreadcrumbs.js

```diff
@@ -16,9 +16,17 @@
 
 import React from "react";
 import { useResources } from "providers/resources";
-import styles from "styles/modules/Resources.module.scss";
+import styles from "styles/modules/Resource.module.scss";
 import Link from "next/link";
 
+const getSearchTermText = (searchTerm) => {
+  if (searchTerm === "all") {
+    return "View all resources";
+  } else {
+    return `"${searchTerm}`;
```

Comment:
Missing a quote here?
```suggestion
    return `"${searchTerm}"`;
```